tutordiscovery/templates/discovery/apps/settings/partials/common.py

```diff
@@ -20,8 +20,14 @@
     }
 }
 
+DISCOVERY_DOCKER_IMAGE_ELASTICSEARCH = "{{ DISCOVERY_DOCKER_IMAGE_ELASTICSEARCH }}"
+DISCOVERY_ELASTICSEARCH_HOST = "{{ DISCOVERY_ELASTICSEARCH_HOST }}"
+DISCOVERY_ELASTICSEARCH_PORT = "{{ DISCOVERY_ELASTICSEARCH_PORT }}"
+DISCOVERY_ELASTICSEARCH_SCHEME = "{{ DISCOVERY_ELASTICSEARCH_SCHEME }}"
+DISCOVERY_ELASTICSEARCH_HEAP_SIZE = "{{ DISCOVERY_ELASTICSEARCH_HEAP_SIZE }}"
```

Comment:
Why are these needed now?

Reply:
As we are running elasticsearch container for/from discovery plugin now, these settings will be need in various patches to run elasticsearch same as it was running through tutor.

Reply:
But if the patches were expecting the config with DISCOVERY_ prefix, won't those patches need to an update as well?

Reply:
yeah, these settings are already being used in all patches in this PR with `DISCOVERY_ prefix`.